### PR TITLE
rails 5.1 helper_method error

### PR DESCRIPTION
**Pull request by sebbean**
following the lead over on devise: https://github.com/plataformatec/devise/pull/3732/files

not sure how to replicate the functionality however

### CODE REVIEW

**Comment by dlitvakb** [comment]
Hey @sebbean I'll take a look into adding the `helper_method` check next week, and get this fixed for good.

**Comment by sebbean** [comment]
I have a PR I can send ur way. Not sure if I have already. Thanks!

**Comment by dlitvakb** [comment]
Sounds good, add those commits to this PR

**Comment by sebbean** [comment]
oops, must not have pushed them

File: contentful_rails.gemspec [hunk 1]
```diff
@@ -17,7 +17,7 @@ Gem::Specification.new do |s|
   s.files = Dir["{app,config,db,lib}/**/*", "MIT-LICENSE", "Rakefile", "README.rdoc"]
   s.test_files = Dir["test/**/*"]
 
-  s.add_dependency "contentful_model", ">= 0.1.7"
+  s.add_dependency "contentful_model", ">= 0.2.0"
```
Comment (dlitvakb) [inline contentful_rails.gemspec]: We should change this to `~> 0.2`

**Pull request merged**